doc/cephfs/snap-schedule.rst

```diff
@@ -148,6 +148,7 @@ Examples::
   ceph fs snap-schedule retention remove / h 24 # remove retention for 24 hourlies
   ceph fs snap-schedule retention add / 24h4w # add 24 hourly and 4 weekly to retention
   ceph fs snap-schedule retention remove / 7d4w # remove 7 daily and 4 weekly, leaves 24 hourly
+  ceph fs snap-schedule retention add / h 55n # retain at most 55 snaps which are an hour apart
```

Comment:
There should also be a note about tuning `mds_max_snaps_per_dir` when retaining more than the default.

Reply:
added a note about `mds_max_snaps_per_dir`